react/tic-tac-toe/src/app/screens/Game/components/MatchHistoryList/components/MatchHistoryEntry/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './styles.module.scss';
+
+function MatchHistoryEntry({ matchEntry }) {
+  return (
+    <div className={styles.matchEntryContainer}>
+      <span className={styles.matchEntry}>
+        {matchEntry.firstPlayer}
```

Comment:
It's always a good idea to use a conditional to check whether or not you receive a value instead of undefined as a prop. That way you you avoid breaking the screen in case you receive undefined as prop. 

```suggestion
        {matchEntry && matchEntry.firstPlayer}
```